docs/contributing/guidelines/working_groups.md

```diff
@@ -0,0 +1,12 @@
+# Working Groups
+
+Mbed OS has a number of established working groups aimed at improving and building new services and features. The working groups bring together like-minded individuals and teams of similar interests and provide a platform to share feedback and discuss what can be delivered as a community.
```

Comment:
Too many 'and's in that last sentence . I would rephrase as follows:
The working groups bring together like-minded individuals and teams of similar interests, providing a platform to share feedback and discuss what can be delivered as a community.

Reply:
Good point, thanks for the suggestion!